src/Writer/SitemapWriter.php

```diff
@@ -16,8 +18,8 @@
  */
 class SitemapWriter implements WriterInterface
 {
-    const LIMIT_SIZE = 10485760;
-    const LIMIT_URL = 50000;
+    public const LIMIT_SIZE = 10485760;
```

Comment:
Can we make this private?

Reply:
That change from public to private would be BC break, but since we are on master it can be done.

I am okay with it, but would prefer to do it in another PR, because that change comes from simply execute `php-cs-fixer`.